src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+
+        System.out.println("Truck machine started its work.");

```

Comment:
The message in the `doWork` method should be more informative and indicate that it is a `Truck` that is working. Consider specifying that explicitly, for example: 'Truck is starting its work.'